src/OrchardCore.Modules/OrchardCore.Tenants/Manifest.cs

```diff
@@ -11,6 +11,10 @@
     Id = "OrchardCore.Tenants",
     Name = "Tenants",
     Description = "Provides a way to manage tenants from the admin.",
+    Dependencies = new[]
+    {
+        "OrchardCore.Settings",
```

Comment:
Why we need settings in Tenants? Any use case require are to add this dependency?

Reply:
I think it’s because `ISiteService` gets registered when OC.Settings is enabled. so if OC.Settings we’re not enabled, `ISiteService` won’t exists yes OC.Tenants depends on it.

Reply:
Right, but I'm not sure if we are using `ISiteService` already used in `OC.Tenants`

Reply:
we are [here](https://github.com/OrchardCMS/OrchardCore/blob/main/src/OrchardCore.Modules/OrchardCore.Tenants/Controllers/AdminController.cs#L58) and [here](https://github.com/OrchardCMS/OrchardCore/blob/main/src/OrchardCore.Modules/OrchardCore.Tenants/Controllers/FeatureProfilesController.cs#L40)

Reply:
I'm wondering how it works before ;)

Reply:
I am guessing because `OC.Settings` is enabled by default with all of the default recipes. will it work if you create a recipe without enabling `OC.Settings`, probably not. once `OC.Settings` is enabled once, it's always enabled.

Reply:
Absolutly right, its always enabled

https://github.com/OrchardCMS/OrchardCore/blob/21c897c26db6336059e93cbd7cdadd91c9b347f7/src/OrchardCore.Modules/OrchardCore.Settings/Manifest.cs#L10

So, I don't think this dependecy is necessary, unless @jtkech has a reason for that

Reply:
@hishamco `IsAlwaysEnabled` will only take effect *after* a the module is actually enabled. This means, if you create a site without enabling `OC.Settings` then it will never be enabled. So for assurance, the dependency to OC.Settings is still needed.

Reply:
@hishamco IsAlwaysEnabled will only take effect after a the module is actually enabled. This means, if you create a site without enabling OC.Settings then it will never be enabled. So for assurance, the dependency to OC.Settings is still needed.

Reply:
Right, but it's enabled by default in almost if not all the recipes

Reply:
Right, but it's enabled by default in almost if not all the recipes

Reply:
what if someone want to use their own recipe and did not enable OC.Settings?

Reply:
I never mind to accept this PR while I already approved ;)